ui/admin/app/router.js

```diff
@@ -10,7 +10,9 @@ Router.map(function () {
   this.route('scopes', function () {
     this.route('scope', { path: ':scope_id' }, function () {
       this.route('authenticate', function () {
-        this.route('method', { path: ':auth_method_id' });
+        this.route('method', { path: ':auth_method_id' }, function() {
+          this.route('oidc');
```

Comment:
Can/should authenticate/method route evolve to handle 'password' route? If we are going to call out oidc maybe we can separate out password route too (although it don't do as much as oidc).

Reply:
The full auth flow for password can be completed on the method route (it's a one-step process).  In the case of OIDC, there's an indefinite polling period while the user authenticates.  This route serves as a kind of waiting room that is only needed for OIDC at this time.